dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlannerBuilder.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Planning.Sequential;
+
+/// <summary>
+/// Builder for <see cref="ISequentialPlanner"/>.
+/// </summary>
+public class SequentialPlannerBuilder
```

Comment:
Do we need a builder for this? Seems heavyweight

Reply:
So, the idea behind `SequentialPlannerBuilder` is to provide easier way for users to initialize sequential planner with instrumentation. Without that, users will be responsible to decorate it by their own, which is not very usable, but still possible. While `SequentialPlannerBuilder` is recommended way for planner initialization, developers will be able to use `SequentialPlanner` directly, which is not going to be instrumented.

Otherwise, for developers, it's going to look like 
```csharp
return new InstrumentedSequentialPlanner(new SequentialPlanner(this._kernel, this._config, this.prompt), this._logger);
```

Reply:
Will replace this builder with extension method for `SequentialPlanner`.